server/views/pages/teams.njk

```diff
@@ -13,7 +13,7 @@
     <thead>
       <tr>
         <th class="govuk-!-width-one-quarter">Name</th>
-        <th class="govuk-!-width-one-quarter"><image src="/assets/images/slack_favicon_32.png" width="32" height="32"></image> Slack (opens app)</th>
+        <th class="govuk-!-width-one-quarter"><img src="/assets/images/slack_favicon_32.png" width="32" height="32" alt="" aria-hidden="true"/> Slack (opens app)</th>
```

Comment:
As mentioned previously, the image doesn't serve an actual purpose, just put it as a background image on the `th` or create a `span` over the text and use it there.